odoo/addons/base/models/ir_ui_view.py

```diff
@@ -390,10 +390,11 @@ def _check_xml(self):
                     view_docs = view_docs[0]
                 for view_arch in view_docs:
                     check = valid_view(view_arch, env=self.env, model=view.model)
+                    view_name = '%s (%s)' % (view.name, view.xml_id) if view.xml_id else view.name
```

Comment:
nitpicking: I always prefer to have explicit parentheses when operator priority is not obvious, as it improves readability, but that's a question of personal taste. Ternary operators are also great and terse for short expressions, but generally decrease readability when the expression is long and includes other operators. Sometimes a good old if/else is better ;)
```suggestion
                    view_name = ('%s (%s)' % (view.name, view.xml_id)) if view.xml_id else view.name
```